tests/test_pro/test_pro_post.py

```diff
@@ -34,10 +38,13 @@ def test_pro_exp(launch_fluent_solver_3ddp_t2):
     )
     solver.execute_tui(r"""/solve/monitors/residual/plot? no """)
     solver.solution.initialization.hybrid_initialize()
-    solver.file.write(file_type="case-data", file_name="out/pro_poly_ini")
+    solver.file.write(
+        file_type="case-data", file_name=os.path.join(out, "pro_poly_ini")
+    )
     solver.mesh.check()
     solver.execute_tui(
-        r"""/file/write-profile "out/test1" symmetry bottom front left right top () pressure velocity-magnitude x-wall-shear pressure-coefficient x-velocity quit """
+        r"""/file/write-profile "%s" symmetry bottom front left right top () pressure velocity-magnitude x-wall-shear pressure-coefficient x-velocity quit """
```

Comment:
Is there a reason you are using ‘execute_tui’ and not the TUI API for all these.  Generally in PyFluent I would have though the latter is the go to if the settings api does not have what you want.

Reply:
Hi @dnwillia-work the plan was to keep everything with settings API and in the place where they were not available I used execute TUI. We keep on revisiting them once settings API for those lines are available and update the tests.

Reply:
OK, I just thought it's better to use the TUI command, `solver.tui`, interface since it's documented right.  `execute_tui` is undocumented.

Reply:
@dnwillia-work yes, that's the point I wanted to make as well.

Reply:
@seanpearsonuk 
These journals were automatically generated using tui to python conversion tool. If we want to use only solver.tui then can development please help updating the conversion tool accordingly?

Reply:
Right.  That explains why it’s like that then.

Reply:
So, you will update this to use `solver.tui`?

Reply:
Hi @dnwillia-work it can be done but it will require some time to update all the places to use solver.tui as we need to discover equivalent commands. 
Also as these tests are also running in FLUTE regression where old journals are being converted with each run so our plan was to use those latest converted journal (whenever available) to update remaining commands in github tests. 
Adding @ypatel-qa if we need to prioritize this work.

Reply:
Hi @dnwillia-work and @ypatel-qa I'm merging this PR for now. If further improvement is needed we can create a new PR.